rustls/src/lib.rs

```diff
@@ -239,14 +239,14 @@
 //! Here's a list of what features are exposed by the rustls crate and what
 //! they mean.
 //!
-//! - `logging`: this makes the rustls crate depend on the `log` crate.
-//!   rustls outputs interesting protocol-level messages at `trace!` and `debug!`
-//!   level, and protocol-level errors at `warn!` and `error!` level.  The log
-//!   messages do not contain secret key data, and so are safe to archive without
-//!   affecting session security.  This feature is in the default set.
-//!
-//! - `tls12`: enables support for TLS version 1.2. This feature is in the default
-//!   set. Note that, due to the additive nature of Cargo features and because it
+//! - `logging`: This feature is in the default set. This makes the rustls crate
```

Comment:
I don't love the capital after ":", and I don't really like having this bit of "metadata" as the first thing mentioned (instead of something related to the actual feature). Maybe do "`logging` (enabled by default): make the rustls crate [..]"?

Reply:
Fixed.